### PR TITLE
change kitsu to osu.direct

### PR DESCRIPTION
**Pull request by HorizonCode**
since Kitsu got rebranded to osu.direct, the url changed

### CODE REVIEW

**Comment by 7ez** [comment]
real osu direct

File: app/objects/beatmap.py [hunk 1]
```diff
@@ -47,8 +47,8 @@ async def api_get_beatmaps(**params: Any) -> Optional[list[dict[str, Any]]]:
         url = "https://old.ppy.sh/api/get_beatmaps"
         params["k"] = str(app.settings.OSU_API_KEY)
     else:
-        # https://doc.kitsu.moe/
-        url = "https://kitsu.moe/api/get_beatmaps"
+        # https://osu.direct/doc
+        url = "https://osu.direct/api/get_beatmaps"
```
Comment (cmyui) [inline app/objects/beatmap.py:51]: is it backwards compatible? seems like https://kitsu.moe/api/get_beatmaps goes to https://osu.direct/home

**Review by cmyui — APPROVED** [review]

File: app/objects/beatmap.py (hunk 1 above)
Comment (cmyui) [reply app/objects/beatmap.py:51]: this should be pretty easy to make 100% backwards compatible with some rerouting rules,
`https://kitsu.moe/$1` -> `https://osu.direct/$1`

Comment (tsunyoku) [reply app/objects/beatmap.py:51]: they seemed to have hard redirected to /home always, it broke akatsuki. this sucks :(

Comment (cmyui) [reply app/objects/beatmap.py:51]: @HorizonCode we recently had this problem when moving from akatsuki.pw to akatsuki.gg

if you're using cloudflare, consider setting up page rules like this for this (and future domain switches)
image

Comment (HorizonCode) [reply app/objects/beatmap.py:51]: i will add that 👍

**Pull request merged**